resources/index.html

```diff
@@ -46,6 +46,9 @@
 		* [Guide to Modern OpenGL Functions](https://github.com/fendevel/Guide-to-Modern-OpenGL-Functions)
 		* [Learn OpenGL](https://learnopengl.com/)
 		* [OpenGL 4.6 Specification](https://www.khronos.org/registry/OpenGL/specs/gl/glspec46.core.pdf)
+		* [OpenGL tutorials by Victor Gordan](https://www.youtube.com/playlist?list=PLPaoO-vpZnumdcb4tZc4x5Q-v7CkrQ6M-)
+		* [Modern OpenGL tutorials by Victor Gordan](https://www.youtube.com/playlist?list=PLPaoO-vpZnukvdD_iY8dCefCV6Jp0hJEv)
```

Comment:
twice the same link, one is enough

Reply:
It's not the same link, it's two different playlist on the same channel. Maybe I could point to the channel directly?

Reply:
Leave out the 2nd, its just 3 videos nd other links provide more modern stuff already, maybe when victor adds more we can add it